hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/OzoneConsts.java

```diff
@@ -533,6 +533,12 @@ public final class OzoneConsts {
    */
   public static final String ETAG = "ETag";
 
+  /**
+   * A constant string used as a separator in various contexts within
+   * the OMDBCheckpoint functions.
+   */
+  public static final String SEPARATOR = "-";
```

Comment:
I think we should be more specific about what separator this is for in global const def.

```suggestion
  /**
   * A constant string used as a separator in various contexts within
   * the OMDBCheckpoint functions.
   */
  public static final String OM_SNAPSHOT_SEPARATOR = "-";
```

Reply:
Done.